lib/publiccloud/provider.pm

```diff
@@ -196,9 +196,10 @@ Creates ~/.ssh/config file with all the common ssh client settings
 
 sub place_ssh_config {
     # configure ssh client
```

Comment:
Could you please expand the comment here explaining what is going one: something like.
"ssh will be configured by a ~/.ssh/config file, the config file come from a template, by default the template is here, the user can overwrite the template like ..., in further steps the template will be edited like this and that, from now on all API like ... will be influenced by content of this file..."

Reply:
Done.